impeller/tessellator/tessellator.h

```diff
@@ -213,6 +212,11 @@ class Tessellator {
   ///
   std::vector<Point> TessellateConvex(const Path& path, Scalar tolerance);
 
+  //----------------------------------------------------------------------------
+  /// @brief      Create a temporary polyline. Only one can exist at a time.
```

Comment:
Per thread? Per process?

Reply:
Per process, tessellator.h is not thread safe.